ptsprojects/zephyr/iutctl.py

```diff
@@ -312,7 +314,7 @@ def _get_reset_cmd_nrf52(self):
         Dependency: nRF5x command line tools
 
         """
-        return 'nrfjprog -f nrf52 -r'
+        return 'nrfjprog -f nrf52 -s {} -r'.format(BotProjects[0]['auto_pts']['board_id'])
```

Comment:
I don't like this change, `BotProjects[0]` can be something else than Zephyr's config. I would prefer to add another parameter to ZephyrCtl.

Reply:
agree, good point.